azure-pipelines.yml

```diff
@@ -28,5 +28,6 @@ jobs:
         PATH="$HOME/.linuxbrew/bin:$HOME/.linuxbrew/sbin:$PATH"
         ln -s "$PWD" "$HOME/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-portable-ruby"
         docker build -f docker/Dockerfile.x86_64 -t homebrew-portable-x86_64 .
+        # TODO: this isn't running the same version as Azure Pipelines have checked out.
```

Comment:
@sjackman I need an assist here: the docker image run isn't quite correct as it's tapping the `master` branch and not the checked out branch in Azure Pipelines (e.g. from the branch, PR or fork). Mind taking a look?

Reply:
I'm happy to. Are you able to wait a few days? I'm on a thesis deadline crunch.

Here's the line of code that taps `Homebrew/portable-ruby`:
https://github.com/Homebrew/homebrew-portable-ruby/blob/2cce4a10d4b0d24585fa9d14a897cd1bffa9273a/docker/Dockerfile.x86_64#L25

I'd suggest not tapping it in the Dockerfile, so that you can instead  tap the appropriate version in `azure-pipelines.yml` after `docker build` and before `brew portable-package`.
https://github.com/Homebrew/homebrew-portable-ruby/blob/2cce4a10d4b0d24585fa9d14a897cd1bffa9273a/azure-pipelines.yml#L30-L31

Reply:
@sjackman Good idea, I'll give that a go.

Reply:
Alternatively, a `docker cp` command could copy the Homebrew/portable-ruby files from Azure to the Docker container.